recipe/meta.yaml

```
@@ -8,47 +8,36 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sqlmodel-{{ version }}.tar.gz
-  sha256: 3371b4d1ad59d2ffd0c530582c2140b6c06b090b32af9b9c6412986d7b117036
+  sha256: cc5c7613c1a5533c9c7867e1aab2fd489a76c9e8a061984da11b4e613c182423
 
 build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
-  noarch: python
-  script:
-    # Unfortunately sqlmodel relies on a custom plugin called "poetry-version-plugin"
-    # which relies on the development version of poetry, which is annoying to install
-    # on conda-forge. But why install some weird plugin when we can just use a simple
-    # Bash hack? ;)
-
-    # Verify that the version is correctly set in __init__.py
-    - grep __version__ sqlmodel/__init__.py | grep {{ version }}
-    # Replace the version number in the pyproject.toml with the one we extracted
-    - sed -i "s/version = \"0\"/version = \"{{ version }}\"/" pyproject.toml
-    # Install
-    - {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<39]
```

```suggestion
  skip: true  # [py<38]
```

https://github.com/fastapi/sqlmodel/blob/0.0.24/pyproject.toml#L10

---

Do we still build for 3.8?

---

We don't still build for it. But no harm in matching upstreams intent.